lambda-src/index.js

```diff
@@ -0,0 +1,50 @@
+const request = require('request-promise-native');
+
+const sendMessage = (message) => {
+    return request.post({
+        url: process.env.WEBHOOK_URL,
+        body: message,
+        json: true,
+    })
+    .then((body) => {
+        if (body === 'ok') {
+            return {};
+        } else {
+            throw new Error(body);
+        }
+    });
+};
+
+const processRecord = (record) => {
+    return sendMessage({
+        text: `<!here>  ${record.Sns.Message}`,
+    });
+};
+
+
+//const handler = (event, context, cb) => {
+exports.handler = (event, context, cb) => {
+    if (!process.env.WEBHOOK_URL) {
+        throw new Error('WEBHOOK_URL environment variable must be defined')
+    }
+    console.log(`event received: ${JSON.stringify(event)}`);
```

Comment:
Do we still want to log this out or was it just for debugging?

Reply:
I'd rather keep it. It's handy know what the triggers were